shared/test/test_channels.rb

```diff
@@ -164,17 +164,28 @@ def test_publish_and_unpublish_channel
     assert_includes result.keys, 'publishedAt'
     assert_nil result['publishedAt']
 
-    # users under age 13 cannot publish projects
+    # users under age 13 cannot publish applab, gamelab or weblab projects,
+    # but can publish artist, playlab, and other project types.
 
     ChannelsApi.any_instance.stubs(:current_user).returns({birthday: 12.years.ago.to_datetime})
-    post "/v3/channels/#{channel_id}/publish/#{project_type}", {}.to_json, 'CONTENT_TYPE' => 'application/json;charset=utf-8'
+
+    post "/v3/channels/#{channel_id}/publish/applab", {}.to_json, 'CONTENT_TYPE' => 'application/json;charset=utf-8'
```

Comment:
It would be nice to break tests of this behavior out into individual test cases instead of adding it to this (already long) test, so that we're obviously covering all the spec'd behavior (where right now we're only checking applab and artist).  The only downside I can think of is if there's some particularly long slow setup step to this test.

* test_student_under_13_can_publish_artist_project
* test_student_under_13_can_publish_playlab_project
* test_student_under_13_cannot_publish_applab_project
* test_student_under_13_cannot_publish_gamelab_project
* test_student_under_13_cannot_publish_weblab_project